include/netlink/route/link/can.h

```diff
@@ -60,6 +60,9 @@ extern int rtnl_link_can_get_data_bittiming(struct rtnl_link *,
 extern int rtnl_link_can_set_data_bittiming(struct rtnl_link *,
                                             const struct can_bittiming *);
 
+int rtnl_link_can_get_device_stats(struct rtnl_link *,
```

Comment:
please prefix the commit message with "route/can: " (and use lower-case for starting the sentence).

Could you also write one sentence what this is about?